developer/15.0/guides/lexical-models/advanced/word-breaker.php

```diff
@@ -230,6 +233,9 @@ class RegExpDerivedSpan implements Span {
 <p>
   <a href="https://github.com/keymanapp/keyman/blob/4211b468949860b8fb4a4707710472ab9e33c581/common/lexical-model-types/index.d.ts#L286-L323">
 The TypeScript definition of <code>WordBreakingFunction</code> and <code>Span</code></a>
+  <a href="./unicode-breaker-extension.md">Extension and customization of the Unicode word-breaker</a>
```

Comment:
```suggestion
  <a href="./unicode-breaker-extension">Extension and customization of the Unicode word-breaker</a>
```

Was getting a "broken link" report from this!

Reply:
Yes, policy on the websites is to exclude both `.md` and `.php` extensions -- bare filenames only. We have some legacy `.php` links but `.md` will intentionally always fail :grin:, so that links are independent of underlying format. (For legacy reasons, links to `.../foo.php` will forward correctly to `.../foo` when we convert `foo.php` to `foo.md`, and then our .htaccess will pass that to the markdown renderer transparently!).